pkg/services/dashboards/service/service.go

```diff
@@ -10,13 +10,9 @@ import (
 func ProvideSimpleDashboardService(
 	features featuremgmt.FeatureToggles,
 	svc *DashboardServiceImpl,
-	k8s k8saccess.K8SAccess,
```

Comment:
should this be the new client?

Reply:
this is restructured in the next branch, so i think it will be easier to handle it there